ci-operator/config/openshift/okd-machine-os/openshift-okd-machine-os-master.yaml

```diff
@@ -19,6 +19,9 @@ images:
   to: okd-rpms
 - dockerfile_path: Dockerfile.generated
   inputs:
+    artifacts:
```

Comment:
I guess the `origin_4.14_artifacts` base image has to be renamed to `artifacts` here too? Same in the v4.14 config

Reply:
Good idea, done